spec/services/deadline_calculator/calendar_months_spec.rb

```diff
@@ -1,159 +1,156 @@
 require 'rails_helper'
 
-
 describe DeadlineCalculator::CalendarMonths do
-    let(:sar)                 { find_or_create :sar_correspondence_type }
-    let(:sar_case)            { create :sar_case }
-    let(:deadline_calculator) { described_class.new sar_case }
-
-    context 'SAR case' do
-      describe '#time_units_desc_for_deadline' do
-        it 'single' do
-          expect(deadline_calculator.time_units_desc_for_deadline)
-            .to eq "calendar month"
-        end
+  let(:sar)                 { find_or_create(:sar_correspondence_type) }
+  let(:sar_case)            { freeze_time { create(:sar_case) } }
```

Comment:
This is the main code change. The other changes are to fix indentation and minor style issues.